addon/components/nrg-search.hbs

```diff
@@ -6,25 +6,37 @@
   {{on-click-outside this.onBlur}}
   ...attributes
 >
-  <div
-    class="ui icon input {{if @disabled 'disabled'}}"
-    role="button"
-    aria-label="Search box"
-    {{on "click" this.onFocus}}
-  >
-    <NrgTextField
-      @focusId={{@focusId}}
-      class="prompt"
-      @disabled={{@disabled}}
-      @placeholder={{this.placeholder}}
-      @readonly={{@readonly}}
-      @model={{this}}
-      @valuePath="displayValue"
-      @onChange={{this.query}}
-    />
-    {{#unless @hideSearchIcon}}
-      <i class="search icon"></i>
-    {{/unless}}
+  <div class="ui label search-container">
+    <div
+      class="ui icon input {{if @disabled 'disabled'}}"
+      role="button"
+      aria-label="Search box"
+      {{on "click" this.onFocus}}
+    >
+      <NrgTextField
+        @focusId={{@focusId}}
+        @class="prompt"
```

Comment:
Did you add the `@` back? It's been deprecated by NRG now:
```suggestion
        class="prompt"
```

Reply:
This was originally on for `4.1.x` but I moved it to the current version and will backport it later. I missed the `@` in the rebase.